docusaurus/docs/reactnative/common-content/core-components/channel/props/on_press_message.mdx

```diff
@@ -11,6 +11,27 @@ You will have to handle these cases when overriding this function.
 | -------- |
 | function |
 
-| Parameter | Description                   |
-| --------- | ----------------------------- |
-| payload   | `{ actionHandlers, message }` |
+| Parameter | Description                                   |
+| --------- | --------------------------------------------- |
+| payload   | `{ additionalInfo, actionHandlers, message }` |
+
+The `additionalInfo` prop is handy for getting information from certain click events.
+A good example of this is getting the user details when a textMention (eg @Enigma-I-am) is clicked.
+
+For example:
+
+```tsx
+    <Channel
+    onPressMessage={({ additionalInfo, emitter }) => {
```

Comment:
`defaultHandler` needs to be imported here from prop

Reply:
✅